velox/connectors/hive/HiveConfig.cpp

```diff
@@ -191,6 +191,22 @@ uint64_t HiveConfig::orcWriterMaxDictionaryMemory(const Config* session) const {
       core::CapacityUnit::BYTE);
 }
 
+bool HiveConfig::orcLinearStripeSizeHeuristics(const Config* session) const {
+  return toCapacity(
+      session->get<std::string>(
+          kOrcLinearStripeSizeHeuristicsSession,
+          config_->get<std::string>(kOrcLinearStripeSizeHeuristics, "true")),
+      core::CapacityUnit::BYTE);
+}
+
+uint64_t HiveConfig::orcMinCompressionSize(const Config* session) const {
```

Comment:
Can we add test to HiveConfigTest for the newly added session property and config?